### DISCUSSION TITLE
What's the maximum CSV size can PandasAI handle?

### DISCUSSION THREAD

**Discussion by eRuaro** [sinaptik-ai/pandas-ai#346] [Q&A] [answered]
Can it handle large CSV files or no? I'm assuming the contents have to fill in the context size of the LLM model being used?

**Comment by mspronesti** [comment, marked as answer]
Pandasai does not work with the file content but with the dataframe schema. It generates the code required for the analysis you want to perform and executes it, but it does not inject the dataframe in the prompt. The concern might arise for dataframes with plenty of columns, but not with the size of the file.
